bin/delete_dependabot_deployment

```diff
@@ -13,6 +13,8 @@ echo "$UAT_RELEASES"
 if [[ $UAT_RELEASES == *"$RELEASE_NAME"* ]]
 then
   helm delete $RELEASE_NAME --namespace=${K8S_NAMESPACE}
+  kubectl --namespace=${K8S_NAMESPACE} delete pvc data-$RELEASE_NAME-postgresql-0
+  kubectl --namespace=${K8S_NAMESPACE} delete pvc redis-data-$RELEASE_NAME-redis-master-0
```

Comment:
edge case handling for redis dependabots potentially - any release name with redis already in it will have `-master-0` not `-redis-master-0`


not tested
```shell
if [[ $RELEASE_NAME == *"redis"* ]]; then
  kubectl --namespace=${K8S_NAMESPACE} delete pvc redis-data-$RELEASE_NAME-master-0
else
  kubectl --namespace=${K8S_NAMESPACE} delete pvc redis-data-$RELEASE_NAME-redis-master-0
fi
```